yoti-sdk-api/src/test/java/com/yoti/api/client/shareurl/policy/DynamicPolicyTest.java

```diff
@@ -306,6 +307,17 @@ public void buildWithIdentityProfile() {
         assertThat(result.getIdentityProfile().getScheme(), is(equalTo(identityProfileScheme)));
     }
 
+    @Test
+    public void buildWithSubject() {
```

Comment:
@irotech I am not really sure to understand why is the subject part of the policy, as it seems to be living in the scenario instead (see [dot-net for example](https://github.com/getyoti/yoti-dotnet-sdk/pull/354/files))